plugins/chat/assets/javascripts/discourse/services/chat-api.js

```diff
@@ -70,17 +70,14 @@ export default class ChatApi extends Service {
   /**
    * Destroys a channel.
    * @param {number} channelId - The ID of the channel.
-   * @param {string} channelName - The name of the channel to be destroyed, used as confirmation.
```

Comment:
I don't understand why we are removing this? It's an important part of the UI

Reply:
Answered here: https://github.com/discourse/discourse/pull/19814#discussion_r1088748042